typings.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module "emoji-unicode" {
+  declare const emojiFn: (emoji: string) => string;
+  export = emojiFn;
+}
+
+declare module "notion-for-each-row" {
```

Comment:
def feel free to kill this dependency btw. it doesn't actually do much besides paginate for me